lib/components/map/default-map.tsx

```diff
@@ -334,7 +336,9 @@ class DefaultMap extends Component {
               case 'park-and-ride':
                 return <ParkAndRideOverlay {...namedLayerProps} />
               case 'stops':
-                return <StopsOverlay {...namedLayerProps} />
+                return (
+                  <StopsOverlay {...namedLayerProps} stopsColor={stopsColor} />
```

Comment:
I don't think you need to explicitly pass in `stopsColor`! `namedLayerProps` already comes from the config, so you can just define it there. This PR is not needed

Reply:
Good catch, although it's not passing through the whole `mapConfig` so I think we will need a pr! I can alternatively add the entire mapColors section of the config to the `namedLayerProps` if you think that would be better

Reply:
You just have to define `stopsColor` as a property of the map layer rather than as a global

Reply:
Gotcha, no you're right! Thanks, I'll close this and adjust accordingly!